### PR TITLE
Update governance, maintainers and create subprojects

### PR DESCRIPTION
**Pull request by justincormack**
As discussed at meetings and here, we are restructuring governance and creating subprojects.
- Notation. Signatures in registries
- TUF. Full TUF registry native implementation

We are also removing the inactive maintainers. We thank them for their contributions and are proud to have them in our alumni.

As this is a governance change it should be approved by 2/3 of maintainers.

### CODE REVIEW

**Comment by diogomonica** [comment]
LGTM

**Comment by ecordell** [comment]
LGTM

**Comment by dlorenc** [comment]
How were these original maintainers chosen? I'm kind of surprised to not see Marina or Brandon Mitchell in the org maintainers, they've been by far the most active for the longest period of time.

File: PROJECTS [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+registry and adds the ability to sign and verify content in the registry. By default it makes
+some different security choices, such as using TOFU by default. As it runs next to a registry
+it is not a standard part of the registry protocol and requires independent storage and for
+clients to have different code to handle signed content.
+
+## Notation
+
+Notation is a project to add signatures as standard items in the registry ecosystem, and to
+build a set of simple tooling for signing and verifying these signatures. This should be
+viewed as similar in security to checking git commit signatures, although the signatures are
```
Comment (dlorenc) [inline PROJECTS:15]: Does this include things like PKI, key management and key discovery? They're typically not part of the git commit model and are expected to be handled out of band.

**Comment by cyli** [comment]
LGTM.  Thanks for the long overdue change!

**Review by cyli — APPROVED** [review]
LGTM!

**Comment by HuKeping** [comment]
LGTM!

**Review by HuKeping — APPROVED** [review]

File: MAINTAINERS [hunk 2]
```diff
@@ -6,21 +6,32 @@
...
 # To extract its contents programmatically, use any TOML-compliant parser.
 #
-# This file is compiled into the MAINTAINERS file in docker/opensource.
-#
 [Org]
-	[Org."Core maintainers"]
+	[Org."Org maintainers"]
+		people = [
+			"justincormack",
+			"niazfk",
```
Comment (JustinCappos) [inline MAINTAINERS:13]: I'd like to suggest a more diverse group.  Having a wider set of people from a larger community for each project should help the group to build broader consensus.

Trishank, Luke, Dan, Marina, and myself would all be interested and I'm sure other community members would welcome the opportunity too.

**Review by endophage — APPROVED** [review]
LGTM

File: PROJECTS [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+generic and can be used for additional purposes.
+
+## TUF
```
Comment (mnm678) [inline PROJECTS:18]: Can we call this `registry-TUF` or `Notary-TUF` or something to reduce the potential for confusion with the original TUF project?

**Comment by justincormack** [comment]
Approved by @justincormack @cyli @diogomonica @HuKeping @endophage @ecordell which is 6/8 maintainers so this passes as 2/3 supermajority as required.

@mnm678 happy to tweak project name afterwards

@JustinCappos we can go ahead and add new maintainers under the new procedures

**Comment by caniszczyk** [comment]
FYI this repo has been moved over to github.com/notaryproject as part of the governance updates

**Comment by NathanMcCauley** [comment]
post-facto LGTM

**Pull request merged**